docs/getting_started/introduction.md

```diff
@@ -0,0 +1,71 @@
+
+
+# What is Time-Series Forecasting?
+
+Generally speaking, forecasting just means making predictions about events in
+the feature. Trivially, while weather forecasting tries to predict the
+weather, in time-series forecasting we want to predict the future values of a
+given time-series.
+
+For example, in electricy production it is very important that demand and
+supply are in balance. Thus, producers anticipate consumer demand for
+electricity and plan production capacity accordingly. In other words, producers
+rely on accurate time-series forecasting of consumer demand for electricity to
+generate just enough supply.
+
+In forecasting, there is the implicit assumption that observable behaviours of
+the past that impact time-series values continue into the future. To stay
+with the electricity example: people will generally consume less energy in the
+night than during the day, will watch TV mostly during the evenings and use
+air conditioners when it's hot during summer.
+
+```{figure} ../_static/electricity-10w.png
+---
+---
+Ten weeks of data plotted over each other -- ``electricity`` dataset.
+```
+
+Naturally, it's impossible to forecast the unpredictable. For instance, in 2019
+it was virtually impossible to account for the possibility of travel
+restrictions due to the Covid-pandemic when trying to forecacst travel demand
+for 2020.
+
+Thus, forecasting operates on the caveat that the underlying factors that
+generate the time-series values don't fundamentally change in the future. It is
+a tool to predict the ordinary and the surprising.
```

Comment:
This seems contradictory with the previous sentence: you shouldn't be able to predict something that is too surprising, given the knowledge on past behaviour, right?